tests/python/contrib/test_ethosn/test_networks.py

```diff
@@ -78,24 +78,16 @@ def get_model():
             )
         return _get_tflite_model(model_path, input_dict, "uint8")
 
-    outputs = []
     inputs = {}
     for input_name in input_dict:
         input_shape = input_dict[input_name]
         inputs[input_name] = tei.get_real_image(input_shape[1], input_shape[2])
 
-    for npu in [False, True]:
-        mod, params = get_model()
-        graph, lib, params = tei.build(
-            mod, params, npu=npu, expected_host_ops=host_ops, npu_partitions=npu_partitions
-        )
-        if npu:
-            tei.assert_lib_hash(lib, compile_hash)
-        if run:
-            outputs.append(tei.run(graph, lib, params, inputs, output_count, npu=npu))
-
+    mod, params = get_model()
+    m = tei.build(mod, params, npu=True, expected_host_ops=host_ops, npu_partitions=npu_partitions)
```

Comment:
After looking into `build`, it seems like it just doesn't go through the Ethos-N partition when `npu=False` but just equivalent to `relay.build`. Accordingly,

- Looks like `npu` will never be False. Do we need to keep this argument?
- If we need to keep this argument, maybe we need better naming such as `with_npu`. `npu` is somewhat confusing.

Reply:
It will never be false for these tests, but it is for others like test_conv2d.

Reply:
I've improved the docs a bit to better explain what npu does.